virtwho/provision/virtwho_host.py

```diff
@@ -296,10 +301,11 @@ def libvirt_pkg_install(ssh):
     Install libvirt related packages.
     :param ssh: ssh access of virt-who host.
     """
-    ssh.runcmd(
-        "yum clean all;"
-        "yum install -y nmap iproute rpcbind libvirt* virt-manager Xorg* gnome*"
-    )
+    # virt-manager pakcage is not readly in rhel-10
```

Comment:
maybe you mean not `ready`?

Reply:
Yes, thanks for the detailed review and edited it.